### PR TITLE
DIART-14 Migrate 'time' folder to Dart

### PR DESCRIPTION
**Pull request by galunecandrey**

### CODE REVIEW

**Review by s0m3chill — APPROVED** [review]

**Review by creageek — APPROVED** [review]

File: di_art/lib/time/Measure.dart [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import 'package:di_art/utils/Pair.dart';
+
+double measureDurationOnly(Function code){
+  int start = DateTime.now().microsecondsSinceEpoch;
+  code();
+  return (DateTime.now().microsecondsSinceEpoch - start) / 1000000.0;
```
Comment (mkovalyk) [inline di_art/lib/time/Measure.dart:21]: As far as I understood you use seconds as your target measure unit, don't you? Do you think it has enough precision?

**Review by mkovalyk — COMMENTED** [review]
I have got an error when I try to run tests: 
Could not find a file named "pubspec.yaml" in "C:\Users\Mike\AppData\Roaming\Pub\Cache\hosted\pub.dartlang.org\shelf_static-0.2.8".
Does anyone have a similar issue?
@s0m3chill @creageek @galunecandrey

File: di_art/lib/time/Measure.dart (hunk 1 above)
Comment (mkovalyk) [reply di_art/lib/time/Measure.dart:21]: I've resolved the issue with compilation :)

Comment (mkovalyk) [reply di_art/lib/time/Measure.dart:21]: But I would rather stick to the more precise solution. It is always possible to come to less precise number but vise versa is not possible :)

Comment (mkovalyk) [reply di_art/lib/time/Measure.dart:21]: @creageek @s0m3chill What do you think?

Comment (galunecandrey) [reply di_art/lib/time/Measure.dart]: This is primarily migration from cocaine. and all values ​​are taken from there.

Comment (galunecandrey) [reply di_art/lib/time/Measure.dart:21]: This is primarily migration from koin. and all values ​​are taken from there.

Comment (galunecandrey) [reply di_art/lib/time/Measure.dart:21]: I am sure that in the future this will make sense in further migration.

**Pull request merged**